docs/4. Options/4.1. Main/⚙️ Min number of characters for trigger.md

```diff
@@ -4,15 +4,22 @@
 - min: `0`
 - max: `10`
 
-**It uses a default value of [[⚙️ Strategy]] if set 0.**
+Note: Setting the value to 0 does not mean the suggestion will be triggered without any inputted character. Depending on the [[⚙️ Strategy]] you choose the actual value will be differ.
 
-| Strategy         | default value |
-| ---------------- | ------------- |
-| [[default]]      | 3             |
-| [[japanese]]     | 2             |
-| [[chinese]]      | 1             |
-| [[arabic]]       | 3             |
-| [[english-only]] | 3             |
+| Strategy         | Actual value |
+| ---------------- | ------------ |
+| [[default]]      | 3            |
+| [[japanese]]     | 2            |
+| [[chinese]]      | 1            |
+| [[arabic]]       | 3            |
+| [[english-only]] | 3            |
+
+This is especially worth to remember if you use [[Custom dictionary complement]]. For example if your dictionary is like this:
```

Comment:
I’m starting to feel it might be a good idea to create a 'FAQ' section🤔. If we put it here, only those who realize something is off with this option will find it, but if it’s in the 'FAQ', more people might find it.

Reply:
I agree. What other FAQs do you have?

Reply:
@ooker777 
No. I need to create the FAQ. Let me merge this PR, I'll move it later.